src/renderer/index.html

```diff
@@ -1,94 +1,97 @@
+        0% {
+          transform: rotate(0);
+          animation-timing-function: cubic-bezier(0.55, 0.055, 0.675, 0.19);
+        }
 
-    @keyframes logo-anim {
-      0% {
-        transform: rotate(0);
-        animation-timing-function: cubic-bezier(0.55, 0.055, 0.675, 0.19);
-      }
+        50% {
+          transform: rotate(-360deg);
+          animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
+        }
 
-      50% {
-        transform: rotate(-360deg);
-        animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
+        100% {
+          transform: rotate(-1080deg);
+        }
       }
 
-      100% {
-        transform: rotate(-1080deg);
+      #react-root {
+        visibility: hidden;
       }
-    }
 
-    #react-root {
-      visibility: hidden;
-    }
-
-    #loader-container {
-      z-index: 99;
-      position: absolute;
-      top: 0;
-      left: 0;
-      right: 0;
-      bottom: 0;
-      display: flex;
-      align-items: center;
-      justify-content: center;
-      perspective: 2000px;
-      transition: opacity 0.8s ease-out;
-      opacity: 0;
-      pointer-events: none;
-    }
+      #loader-container {
+        z-index: 99;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        perspective: 2000px;
+        transition: opacity 0.8s ease-out;
+        opacity: 0;
+        pointer-events: none;
+      }
 
-    #loader-container.loading {
-      opacity: 1;
-    }
+      #loader-container.loading {
+        opacity: 1;
+      }
 
-    #loader {
-      width: 70px;
-      height: 70px;
-      background-color: white;
-      border-radius: 50%;
-      padding: 12px;
-      box-sizing: content-box;
-    }
+      #loader {
+        width: 70px;
+        height: 70px;
+        background-color: white;
+        border-radius: 50%;
+        padding: 12px;
+        box-sizing: content-box;
+      }
 
-    #loader.loaded {
-      animation: loaded-anim 0.7s cubic-bezier(0.55, 0.085, 0.68, 0.53) both;
-    }
+      #loader.loaded {
+        animation: loaded-anim 0.7s cubic-bezier(0.55, 0.085, 0.68, 0.53) both;
+      }
 
-    #logo {
-      animation: logo-anim 1.8s infinite;
-      height: 100%;
-      width: 100%;
-    }
-  </style>
-</head>
+      #logo {
+        animation: logo-anim 1.8s infinite;
+        height: 100%;
+        width: 100%;
+        user-select: none;
+        pointer-events: none;
+      }
+    </style>
+  </head>
 
-<body>
-  <div id="react-root"></div>
-  <div id="loader-container">
-    <div id="loader">
-      <img id="logo" />
+  <body>
+    <div id="react-root"></div>
+    <div id="loader-container">
+      <div id="loader">
+        <img id="logo" />
+      </div>
     </div>
-  </div>
-  <div id="modals"></div>
-</body>
-
-</html>
\ No newline at end of file
+    <div id="modals"></div>
+  </body>
+</html>
```

Comment:
please undo this file change, it is not related to the commit. (if it is needed, should be another commit)